scripts/release/prepare-release.sh

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2025 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+# Automated release script for Jaeger
+# This script automates the manual steps in the release process
+
+set -euo pipefail
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Configuration
+REPO="jaegertracing/jaeger"
+DRY_RUN=false
+AUTO_TAG=false
+
+# Functions
+log_info() {
+    echo -e "${BLUE}[INFO]${NC} $1"
+}
+
+log_success() {
+    echo -e "${GREEN}[SUCCESS]${NC} $1"
+}
+
+log_warning() {
+    echo -e "${YELLOW}[WARNING]${NC} $1"
+}
+
+log_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Automated release script for Jaeger
+
+OPTIONS:
+    -d, --dry-run     Run in dry-run mode (no actual changes)
+    -t, --auto-tag    Automatically create and push tags
```

Comment:
the tags in the current instructions are created after the PR is merged. How can they be in this script?